packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -397,10 +397,187 @@ TileMoreOptions.displayName = 'Tile.MoreOptions';
 const TileButtons = polymorphic('iui-tile-buttons');
 TileButtons.displayName = 'Tile.Buttons';
 
+// ----------------------------------------------------------------------------
+type TileLegacyProps = {
+  /**
+   * Name or title of the tile.
+   */
+  name: React.ReactNode;
+  /**
+   * Description text of the tile.
+   * Gets truncated if it can't fit in the tile.
+   */
+  description?: React.ReactNode;
+  /**
+   * Metadata section located below description.
+   * @example
+   * <Tile
+   *  // ...
+   *  metadata='basic metadata'
+   *  // or
+   *  metadata={<span><SvgClock /> 2021-01-01, 04:30 AM</span>}
+   *  // or
+   *  metadata={<>
+   *    <SvgTag2 />
+   *    <TagContainer><Tag variant='basic'>Tag 1</Tag><Tag variant='basic'>Tag 2</Tag></TagContainer>
+   *  </>}
+   * />
+   */
+  metadata?: React.ReactNode;
+  /**
+   * Thumbnail image url, a custom component or an svg.
+   * @example
+   * <Tile
+   *  // ...
+   *  thumbnail='/url/to/image.jpg'
+   *  // or
+   *  thumbnail={<Avatar image={<img src='icon.png' />} />}
+   *  // or
+   *  thumbnail={<SvgImodelHollow />}
+   * />
+   */
+  thumbnail?: string | React.ReactNode;
+  /**
+   * `Badge` shown on the bottom right of thumbnail.
+   */
+  badge?: React.ReactNode;
+  /**
+   * Icon shown on top left of the tile. Also known as "type indicator".
+   * Recommended to use an invisible `IconButton`.
+   */
+  leftIcon?: React.ReactNode;
+  /**
+   * Icon shown on top right of the tile. Also known as "quick action".
+   * Recommended to use an invisible `IconButton`.
+   */
+  rightIcon?: React.ReactNode;
+  /**
+   * Upto two buttons shown at the bottom of the tile.
+   */
+  buttons?: [React.ReactNode?, React.ReactNode?];
+  /**
+   * Dropdown menu containing `MenuItem`s.
+   */
+  moreOptions?: React.ReactNode[];
+  /**
+   * Status of the tile.
+   */
+  status?: 'positive' | 'warning' | 'negative';
+  /**
+   * Whether the tile is selected or in "active" state.
+   * Gets highlighted and shows a checkmark icon near tile name.
+   */
+  isSelected?: boolean;
+  /**
+   * Whether the tile is "new". Tile name becomes bold and gets a new status icon.
+   */
+  isNew?: boolean;
+  /**
+   * Default tile variant or the folder layout.
+   * @default 'default'
+   */
+  variant?: 'default' | 'folder';
+  /**
+   * Any custom nodes that will be appended to the tile's main content.
+   */
+  children?: React.ReactNode;
+  /**
+   * Whether the tile is expected to be interactable (i.e. `onClick`).
+   * It becomes focusable and gets on hover styling.
+   */
+  isActionable?: boolean;
+  /**
+   * Display a loading state.
+   * @default false
+   */
+  isLoading?: boolean;
+  /**
+   * Flag whether the tile is disabled.
+   *
+   * Note: This only affects the tile. You need to manually disable
+   * the buttons and other interactive elements inside the tile.
+   *
+   * @default false
+   */
+  isDisabled?: boolean;
+  onClick?: React.MouseEventHandler<HTMLElement>;
```

Comment:
add jsdoc here? it's not obvious that this onClick is not applied on the wrapper